.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "browsers": ["last 2 versions", "safari >= 7", "ie >= 9"]
```

Comment:
@weihanglo I made a rough (very rough) attempt at browserlist in #419, although the more I stare at that file, the more shame I feel and it probably needs to be refactored. I like your approach better than my sad approach of listing browsers I didn't want to see warnings from in stylelint output.